PyDSS/utils/simulation_utils.py

```diff
@@ -0,0 +1,20 @@
+
+import math
+
+from PyDSS.value_storage import ValueContainer, ValueByNumber
+
+
+def calculate_line_loading_percent(line):
+    normal_amps = line.GetValue("NormalAmps", convert=True).value
+    currents = line.GetValue("Currents", convert=True).value
+    current = max([math.sqrt(x.real**2 + x.imag**2) for x in currents])
+    loading = current / normal_amps * 100
+    return ValueByNumber(line.Name, "LineLoading", loading)
+
+
+def calculate_transformer_loading_percent(transformer):
+    normal_amps = transformer.GetValue("NormalAmps", convert=True).value
+    currents = transformer.GetValue("Currents", convert=True).value
+    current = max([math.sqrt(x.real**2 + x.imag**2) for x in currents])
+    loading = current / normal_amps * 100
+    return ValueByNumber(transformer.Name, "TransformerLoading", loading)
```

Comment:
@akshaykj92 Can you review these two functions?  I'm not positive that the transformer function is correct.

Here's how this process will work:
- The user adds Line.LoadingPercent as an entry in Exports.toml with optional limits.  Example:  https://github.com/NREL/PyDSS/pull/23/files#diff-3f5f30c7d0e515e2df873e0f4e6ed53fR67
- This dictionary maps that key to this `calculate_line_loading_percent` function:  https://github.com/NREL/PyDSS/pull/23/files#diff-b60a2cfe536928acb4bf8d44f3b9df68R17
- PyDSS calls this function every step of the simulation and stores all values that match the criteria in an HDF file.
- You can programmatically access a dataframe of the violations after the simulation.  PyDSS can optionally export that dataframe to a CSV file.

Reply:
Looks good